src/Task/FileSystem/CleanDir.php

```diff
@@ -16,8 +17,13 @@
  */
 class CleanDir extends BaseDir
 {
+    use FileAndFolderCheck;
+
     public function run()
     {
+        if (!$this->checkResources($this->dirs, 'dir')) {
+            return Result::error($this, 'Resources are missing!');
```

Comment:
For this specific case the error should state like "Source directories are missing"